harbi.py

```diff
@@ -239,16 +239,21 @@ def change_username(session_id, device_id, iid, last_username, new_username):
         return "Failed to change username: " + str(result)
 
 
-def main():
+def main(session_id=0): # Retry if requested username doesnt work
     device_id = str(random.randint(777777788, 999999999999))
     iid = str(random.randint(777777788, 999999999999))
-    session_id = input("Enter the sessionid : ")

```

Comment:
if you mean to store session id then code trying usernames until you got username you can edit
line 273
it should be "        return "Failed to change username: " + str(result)"

replace it with:
        device_id = str(random.randint(777777788, 999999999999))
        iid = str(random.randint(777777788, 999999999999))
        new_username = input("Enter the new username you wish to set: ")
        print(change_username(session_id, device_id, iid, last_username, new_username))